docker/hq-compose.yml

```diff
@@ -159,7 +147,6 @@ services:
   citus_manager:
     image: 'citusdata/membership-manager:0.2.0'
     volumes: ["${DOCKER_SOCK:-/var/run/docker.sock}:/var/run/docker.sock"]
-    depends_on: { citus_master: { condition: service_healthy } }
```

Comment:
you can't extend a service that has `depend_on` declared. So this works here because we don't extend citus - we have a separate compose file that get's used if you want to run it locally.